connect/src/main/java/com/stripe/android/connect/AccountOnboardingView.kt

```diff
@@ -1,30 +1,37 @@
 package com.stripe.android.connect
 
 import android.content.Context
+import android.os.Parcelable
 import android.util.AttributeSet
 import android.widget.FrameLayout
 import androidx.annotation.RestrictTo
+import androidx.core.content.withStyledAttributes
 import com.stripe.android.connect.webview.StripeConnectWebViewContainer
 import com.stripe.android.connect.webview.StripeConnectWebViewContainerImpl
 import com.stripe.android.connect.webview.serialization.SetOnExit
 import com.stripe.android.connect.webview.serialization.SetterFunctionCalledMessage
+import com.stripe.android.connect.webview.serialization.toJs
+import com.stripe.android.connect.webview.serialization.toJsonObject
+import dev.drewhamilton.poko.Poko
+import kotlinx.parcelize.Parcelize
 
 @PrivateBetaConnectSDK
-@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class AccountOnboardingView private constructor(
```

Comment:
👀 Cannot restrict this class otherwise it's not accessible to users in XML layouts